turbopack/crates/turbopack-core/src/module_graph/mod.rs

```diff
@@ -1573,13 +1212,16 @@ impl ModuleGraphRef {
         entries: impl IntoIterator<Item = (ResolvedVc<Box<dyn Module>>, P)>,
         state: &mut S,
         mut visit: impl FnMut(
-            Option<(&'_ SingleModuleGraphModuleNode, &'_ RefData)>,
-            &'_ SingleModuleGraphModuleNode,
+            Option<(ResolvedVc<Box<dyn Module>>, &'_ RefData)>,
+            ResolvedVc<Box<dyn Module>>,
             &mut S,
         ) -> Result<GraphTraversalAction>,
-        priority: impl Fn(&'_ SingleModuleGraphModuleNode, &mut S) -> Result<P>,
+        priority: impl Fn(ResolvedVc<Box<dyn Module>>, &mut S) -> Result<P>,
     ) -> Result<usize> {
         let graphs = &self.graphs;
+        if self.skip_visited_module_children {
+            bail!("traverse_edges_fixed_point_with_priority musn't be called on individual graphs");
```

Comment:
panic? this would be a programming error, not an eventual consistency issue, right?